### ISSUE TITLE
[Drop a pin] Instruction popup does not appear

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#2315]
![image](https://github.com/google/ground-android/assets/228050/4c160f7b-1c54-46a1-a1b8-274887f45310)

**Comment by gino-m** [comment]
@sufyanAbbasi I think this is WIP, is that right?

**Comment by sufyanAbbasi** [comment]
Yes, currently this UI is only an icon in the top right which needs to be replaced by the same system that we use for the draw area instruction. I'll make sure to get it working today!
